types/album-record.ts

```diff
@@ -0,0 +1,10 @@
+type ImageRecordType = {
+  id: string
+  filepath: string
+  labels: string
```

Comment:
Are these files added by default? Just wondering why they changed? Did you remember to merge/rebase develop?

Reply:
No, I think I added this one for some reason, but it's not used anywhere so I've removed it